src/raiden_libs/matrix.py

```diff
@@ -274,3 +272,51 @@ def _handle_message(self, room: Any, event: dict) -> bool:
             self.message_received_callback(message)
 
         return True
+
+    def join_global_rooms(
+        self, client: GMatrixClient, available_servers: Sequence[str] = ()
+    ) -> None:
+        """Join or create a global public room with given name on all available servers.
+        If can't, create a public room with the name on each server.
```

Comment:
nitpick: If can't -> If not possible  ... or If you can't. Also "with the name" to "with the given name"